### PR TITLE
Feature/22280 add application insights library

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: admin/config.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const getEnvironment = () => {
 module.exports = {
   AZURE_STORAGE_CONNECTION_STRING: process.env.AZURE_STORAGE_CONNECTION_STRING,
   GOOGLE_TRACKING_ID: process.env.GOOGLE_TRACKING_ID,
+  APP_INSIGHTS_CLIENT_KEY: process.env.APP_INSIGHTS_CLIENT_KEY,
```
Comment (GuyHarwood) [inline admin/config.js]: This is already available as `APPINSIGHTS_INSTRUMENTATIONKEY`

File: admin/helpers/browserSecurity.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const config = require('../config')
 const init = (app) => {
   app.use(helmet())
   app.use(helmet.noCache())
-  const scriptSources = ["'self'", "'unsafe-inline'", 'https://www.google-analytics.com', 'https://www.googletagmanager.com']
+  const scriptSources = ["'self'", "'unsafe-inline'", 'https://www.google-analytics.com', 'https://www.googletagmanager.com', 'https://az416426.vo.msecnd.net']
```
Comment (GuyHarwood) [inline admin/helpers/browserSecurity.js]: hmm, i wonder if they might change this random looking URL 🤔 hope not 😄 could be tricky for us

File: admin/helpers/index.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ module.exports = function (app) {
   app.locals.globalHeaderText = 'GOV.UK'
   app.locals.crownCopyrightMessage = null
   app.locals.googleTrackingId = config.GOOGLE_TRACKING_ID
+  app.locals.appInsightsClientKey = config.APP_INSIGHTS_CLIENT_KEY
```
Comment (GuyHarwood) [inline admin/helpers/index.js]: same as comment above

File: admin/helpers/browserSecurity.js (hunk 2 above)
Comment (GeorgeChatzigiannis) [reply admin/helpers/browserSecurity.js]: this looks indeed like it was generated and likely to change.

**Pull request merged**